Translated/Tasks/Side/Dughal/ss0030741701.csv

```diff
@@ -0,0 +1,4 @@
+Title#0,"""Dighal's Mining Guide-Ⅰ"""
```

Comment:
Please remove `-Ⅰ`.  We use roman numerals, and not the full width version that the japanese use. You must type it out, i.e. `Ⅰ` becomes `I`. We don't add hyphens either.